lib/Frontend/CompilerInvocation.cpp

```diff
@@ -10,9 +10,11 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "swift/Frontend/Frontend.h"
```

Comment:
I think this one can stay, since it's the main header for the file despite not having the same name.